### ISSUE TITLE
When using Netlify Functions, throwing error

### ISSUE THREAD

**Issue by dalisoft** [SkeLLLa/fastify-oas#6]
Hello and thanks.
Here some logs
```bash
Error during invocation:  TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:107:11)
    at Object.dirname (path.js:1121:5)
    at resolve (webpack-internal:///../node_modules/app-root-path/lib/resolve.js:82:22)
    at module.exports (webpack-internal:///../node_modules/app-root-path/lib/app-root-path.js:6:20)
    at Object.eval (webpack-internal:///../node_modules/app-root-path/index.js:4:18)
    at eval (webpack-internal:///../node_modules/app-root-path/index.js:5:30)
    at Object.../node_modules/app-root-path/index.js (/Users/dalisoft/Desktop/Dev Env from null/fastify-dev-env/functions-build/server.js:3569:1)
    at __webpack_require__ (/Users/dalisoft/Desktop/Dev Env from null/fastify-dev-env/functions-build/server.js:20:30)
    at eval (webpack-internal:///../node_modules/fastify-oas/lib/openapi/constructor.js:4:14)
    at Object.../node_modules/fastify-oas/lib/openapi/constructor.js (/Users/dalisoft/Desktop/Dev Env from null/fastify-dev-env/functions-build/server.js:5263:1)
    at __webpack_require__ (/Users/dalisoft/Desktop/Dev Env from null/fastify-dev-env/functions-build/server.js:20:30)
    at eval (webpack-internal:///../node_modules/fastify-oas/lib/openapi/index.js:1:21)
    at Object.../node_modules/fastify-oas/lib/openapi/index.js (/Users/dalisoft/Desktop/Dev Env from null/fastify-dev-env/functions-build/server.js:5274:1)
    at __webpack_require__ (/Users/dalisoft/Desktop/Dev Env from null/fastify-dev-env/functions-build/server.js:20:30)
    at eval (webpack-internal:///../node_modules/fastify-oas/lib/index.js:2:17)
    at Object.../node_modules/fastify-oas/lib/index.js (/Users/dalisoft/Desktop/Dev Env from null/fastify-dev-env/functions-build/server.js:5252:1)
```

**Comment by dalisoft** [comment]
Repo lives [here](https://github.com/dalisoft/fastify-dev-env)

**Comment by SkeLLLa** [comment]
Hi. As far as you can see in stacktrace the root cause of error is in [app-root-path](https://github.com/inxilpro/node-app-root-path) module
`resolve (webpack-internal:///../node_modules/app-root-path/lib/resolve.js:82:22)` 
Here's the file https://github.com/inxilpro/node-app-root-path/blob/master/lib/resolve.js#L82

My guess is that webpack or one of it's loaders doing some evil eval magic and doesn't respect node's api. That's causing an error whenever you `require('app-root-path')`.

So basically it's an issue with webpack and app-root-path module.

**Comment by dalisoft** [comment]
Thanks, yep, you right. APP_ROOT_PATH=/ fixes this :)

**Comment by SkeLLLa** [comment]
Great. So I'm closing this for now as resolved. Feel free to reopen it if you need.
